scripts/migrate.js

```diff
@@ -0,0 +1,13 @@
+const Taskflow = require('@ukhomeoffice/taskflow');
+const config = require('../config');
+
+const migrate = () => {
+  return Promise.resolve()
+    .then(() => Taskflow({ db: config.taskflowDB }).migrate())
+    .catch(err => {
+      console.log(err);
+      process.exit(1);
+    });
+};
+
+migrate();
```

Comment:
If I were being pernickety I'd point out that there's no need to wrap code that's executed exactly once in a function. Good job I'm not.